Integrations/SecBI/SecBI.py

```diff
@@ -0,0 +1,197 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+''' IMPORTS '''
+
+import json
+import requests
+from distutils.util import strtobool
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS/PARAMS '''
+SECBI_API_ROOT = '/api/v1'
+SECBI_API_ENDPOINT_STATUS = SECBI_API_ROOT + '/status'
+SECBI_API_ENDPOINT_INCIDENT = SECBI_API_ROOT + '/thirdparties/integration/incident'
+SECBI_API_ENDPOINT_INCIDENTS = SECBI_API_ROOT + '/thirdparties/integration/incidents'
+SECBI_INCIDENT_FIELDS = ['id', 'host', 'identity', 'internalIp', 'sIp', 'firstAppearance', 'lastAppearance']
+
+''' HELPER FUNCTIONS '''
+
+
+def get_dict_without_none(**kwargs):
+    d = {k: v for k, v in kwargs.items() if v is not None}
+    if len(d) == 0:
+        return None
+    return d
+
+
+''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+class SecBIIncident(object):
+    def __init__(self, incident_json: dict) -> None:
+        super().__init__()
+        for field in SECBI_INCIDENT_FIELDS:
+            setattr(self, field, incident_json[field])
+
+    def __str__(self) -> str:
+        return json.dumps(self.__dict__)
+
+    def __repr__(self) -> str:
+        return '{} ({})'.format(self.__class__.__name__, self.__str__())
+
+
+class SecBIClient(BaseClient):
+
+    def secbi_get_incidents_list(self, query, limit):
+        """
+        SecBI Get All Incident IDs
+        :param query: The Query by which to filter the Incident IDs
+        :param limit: Limit amount of IDs to return (-1) for all
+        :return: SecBI Incident IDs List
+        """
+        params = get_dict_without_none(query=query, limit=limit)
+        LOG('Performing SecBI get_incidents_list request to {}/{} with params={}'.format(self._base_url,
+                                                                                         SECBI_API_ENDPOINT_INCIDENTS,
+                                                                                         str(params)))
+        try:
+            ids_list_str = self._http_request(method='GET',
+                                              url_suffix=SECBI_API_ENDPOINT_INCIDENTS,
+                                              params=params,
+                                              resp_type='text')
+        except Exception as e:
+            LOG("Error when fetching Incidents list from SecBI: {}".format(e.args[0]))
+            LOG.print_log()
+            raise
+        return json.loads(ids_list_str)
```

Comment:
Why not just send `resp_type=json` if we're doing here `json.loads()` here anyway ?